compose/bin/setup

```diff
@@ -7,7 +7,8 @@ bin/stop
 sed -e 's/5f5929ef9f2ec4ca048a2add261d22c92807630f/ce31e720d60451784b9fdb3769e43e149f50d436/' src/composer.lock > composer.lock && mv composer.lock src/composer.lock
 
 docker-compose -f docker-compose.yml up -d
-sleep 1 #Ensure containers are started...
+[ $? != 0 ] && echo "Failed to start Docker services" && exit
+sleep 5 #Ensure containers are started...
```

Comment:
it's minute, but i think we should change this to `sleep 1` instead of `sleep 5`?

Reply:
When I to start docker in the first time in VM (slower machine), the one second is not enough. But only when the problem occurs for the first time.
So I chose 5 seconds to ensure containers are started... I think 3 seconds also work for most slower machines.